### DISCUSSION TITLE
How do you prevent similar replies to unrelated questions in few-shot?

### DISCUSSION THREAD

**Discussion by niansa** [ggml-org/llama.cpp#606] [Q&A]
Hey!

I keep on observing this behavior:
![image](https://user-images.githubusercontent.com/20720267/228651711-048fb49b-c417-408f-af4e-8023c424caaf.png)

What is the best or most common way to prevent this from happening? Higher temperature? Repeat penalty? Both? Or a longer prompt (mine already is pretty long tho)?

Thanks in advance
Niansa

**Comment by ggerganov** [comment]
If you get similar replies, you should immediately start running.
(sorry 😄 )
